### DISCUSSION TITLE
Can Bluetooth connection parameters be modified in ESP32 A2DP Sink Mode?

### DISCUSSION THREAD

**Discussion by litezxc** [pschatzmann/ESP32-A2DP#637] [Q&A] [answered]
Hello,

I am currently working with the ESP32 as a Bluetooth A2DP sink, outputting audio to an external DAC. I would like to know if it is possible to modify the Bluetooth connection parameters while the ESP32 is in this sink mode. Specifically, I am constrained to using a standard format of 44100 Hz, 16 bits, and 2 channels for audio output. 

However, I have noticed that codec supports a sample rate of 48000 Hz. Is there a way to set this sample rate or other additional parameters such as bitrate, number of subbands, etc.?

Thank you for any insights you can provide!

**Comment by pschatzmann** [comment]
You will need to check if it is supported by your A2DP source to change e.g. the rate

**Reply by pschatzmann** [reply, marked as answer]
Please check the [IDF documentation](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-reference/bluetooth/esp_a2dp.html): to my knowledge there is no such thing...
